Sofa/framework/Helper/src/sofa/helper/LCPcalc.cpp

```diff
@@ -1821,7 +1839,7 @@ int nlcp_gaussseidelTimed(int dim, SReal *dfree, SReal**W, SReal*f, SReal mu, SR
                     delete W33[i];
                 free(W33);
                 //printf("Convergence after %d iteration(s) with tolerance : %f and error : %f with dim : %d\n",it, tol, error, dim);
-                //afficheLCP(dfree,W,f,dim);
+                //printLCP(dfree,W,f,dim);
```

Comment:
cannot suggest (deleted line) but I guess we should either remove the comment or uncomment with msg_inf()

Reply:
We are in a free function, so I guess this comment is for debug purposes. Let's delete it